src/components/Insights.js

```diff
@@ -0,0 +1,58 @@
+import Chart from 'react-apexcharts'
+import './Insights.css'
+
+export const Insights = () => {
+    const chartmake = JSON.parse(localStorage.getItem(localStorage.getItem('email') + 'movies'))
+    return (
+        <div className='whole-insights'>
+            <div className='heading-style'>
+                <div className='heading'>Bar Chart Developed By Movies Suggestion Team</div>
+            </div>
+            <div className='chart'>
+                <Chart
+                    type='bar'
+                    width={1380}
+                    heigth={1000}
+                    series={[
```

Comment:
take out series config from html block and place it inside the component variable and use it here.